src/uct/api/v2/uct_v2.h

```diff
@@ -1068,7 +1103,7 @@ ucs_status_t uct_ep_query(uct_ep_h ep, uct_ep_attr_t *ep_attr);
  * @return Nonzero if reachable, 0 if not.
  */
 int uct_iface_is_reachable_v2(uct_iface_h iface,
-                              const uct_iface_is_reachable_params_t *params);
+                              uct_iface_is_reachable_params_t *params);
```

Comment:
We need to add unit tests to `gtest\uct` to test the API. Currently we don't have such tests at all. We only cover this API in `gtest\ucp\test_ucp_wireup.cc`, `gtest\uct\ib\test_ib_pkey.cc`. However, this is not enough. We need to test the exact API using different input parameters. I think adding a simple one at least for new fields would be good way to start.